sklearn/metrics/_classification.py

```diff
@@ -776,6 +778,8 @@ def matthews_corrcoef(y_true, y_pred, sample_weight=None):
     sample_weight : array-like of shape (n_samples,), default=None
         Sample weights.
 
+    .. versionadded:: 0.18    
```

Comment:
I think if this is for the sample_weight, you need and extra indent. Check the rendered docs and the other instances as well.

Reply:
Done! All issues should be fixed now.